terraform/bootstrap/imports.tf

```diff
@@ -0,0 +1,10 @@
+# The notify-management space contains resources that don't belong in our
+# deployed environment spaces (notify-sandbox, etc.) including:
+#   * the bucket that holds Terraform state
+#   * supplemental service brokers, letting cloud.gov provision AWS services
+#   * some service accounts used for deployment
+
+import {
+  to = cloudfoundry_space.notify-management
+  id = "a022fabe-056c-4329-9d4f-3f7a67442d36"
+}
```

Comment:
Calling out the fact that with the way this is done, which is very straightforward and makes sense, I'm assuming this'll be the approach for the other spaces that aren't currently under Terraform management (e.g., `notify-staging`)?

Reply:
Yup! This is the technique I think we should use to bring other spaces into Terraform